uniffi_bindgen/src/bindings/swift/gen_swift/mod.rs

```diff
@@ -526,22 +533,24 @@ pub mod filters {
 
     /// Get the idiomatic Swift rendering of a class name (for enums, records, errors, etc).
     pub fn class_name(nm: &dyn fmt::Display) -> Result<String, askama::Error> {
-        Ok(oracle().class_name(nm))
+        Ok(escape_reserved_word_identifier(oracle().class_name(nm)))
```

Comment:
My general feeling is that these wrappers would be better in the `KotlinCodeOracle` class.  That way we would get the same results when we make the calls from the Rust code and the template code.  But I don't have that strong of an opinion.  If there's a good reason to put them in the filters instead then that's fine with me.